app/javascript/controllers/file_auth_controller.js

```diff
@@ -157,6 +160,12 @@ export default class extends Controller {
       })
   }
 
+  requiresLogin(authResponse) {
+    if (authResponse.status == 401 && authResponse.heading.en[0].includes('log in')){
+      window.dispatchEvent(new CustomEvent('login-success', { detail: {} }))
```

Comment:
Should be a new event type? This is not occurring after they have successfully logged in. This is after authorization.  We're conflating meanings vis-à-vis authN and authZ.

Reply:
I was unsure about this, because this is being called after the application grabs the local token and checks the access which means the user kind of logged in but the application did it for them?

Reply:
Can we get rid of the existing `login-success` (in `addPostCallbackListener`) and just have this new event?

Reply:
so instead of login-success something like show logged in banner or something along those lines?

Reply:
I went with show-message-panel